### PR TITLE
review of azure hosting content

### PR DESCRIPTION
**Pull request by yvesgoeleven**
Review as requested for

https://github.com/Particular/docs.particular.net/issues/2069
https://github.com/Particular/docs.particular.net/issues/2067
https://github.com/Particular/docs.particular.net/issues/2066

### CODE REVIEW

**Comment by yvesgoeleven** [comment]
@Particular/docs-maintainers @Particular/azure-host-maintainers  this one is ready for review

File: nservicebus/hosting/cloud-services-host/faq.md [hunk 1]
```diff
@@ -26,4 +27,16 @@ This is almost always related to an exception happening at startup of the RoleEn
 
 ### Exceptions occurring at startup are not visible in the logs
 
-When using the diagnostics service in Cloud Services, this service starts in parallel with the startup code. If an exception occurs at this point in time, the code may not be able to call the diagnostics service yet and the exception information may get lost. Use [IntelliTrace](https://msdn.microsoft.com/en-us/library/dd264915.aspx) and [Historical Debugging](https://msdn.microsoft.com/en-us/library/mt228143.aspx) instead to learn more about the cause of the exception.
\ No newline at end of file
+When using the diagnostics service in Cloud Services, this service starts in parallel with the startup code. If an exception occurs at this point in time, the code may not be able to call the diagnostics service yet and the exception information may get lost. Use [IntelliTrace](https://msdn.microsoft.com/en-us/library/dd264915.aspx) and [Historical Debugging](https://msdn.microsoft.com/en-us/library/mt228143.aspx) instead to learn more about the cause of the exception.
+
```
Comment (weralabaj) [inline nservicebus/hosting/cloud-services-host/faq.md]: there should be double empty lines before each section header

**Comment by weralabaj** [comment]
@yvesgoeleven LGTM, do you want to wait for @Particular/azure-host-maintainers to have a look to? if not necessary I can merge right away

**Comment by yvesgoeleven** [comment]
@weralabaj it should be good imo, feel free to merge

File: nservicebus/hosting/cloud-services-host/index_web_cloudserviceshost_[7,).partial.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+A short explanation of each:
+
+ *  Logs will be sent to the `Trace` infrastructure, which should have been configured with Azure diagnostic monitor trace listener by the Visual Studio tooling.
+ * `UseTransport<AzureStorageQueueTransport>`: Sets [Azure storage queues](/nservicebus/azure-storage-queues/) as the [transport](/nservicebus/transports).
```
Comment (SeanFeldman) [inline nservicebus/hosting/cloud-services-host/index_web_cloudserviceshost_[7,).partial.md]: Biased samples 😉 Should one configure with ASB?

**Pull request merged**